src/components/organisms/Navbar/NavbarDropdown/NavbarDropdown.tsx

```diff
@@ -148,7 +149,9 @@ export default class NavbarDropdown extends React.Component<NavbarDropdownProps,
           {label}
         </NavbarLink>
         <NavbarDropdownListContainer open={open}>
-          <NavbarDropdownList aria-label={label}>
+          {/* To avoid an undefined aria-label the fallback to label was added when aria-label property is not defined */}
+          {/* TODO remove fallback to label */}
```

Comment:
I would make a GH issue for this so we can track the progress of fixing the issue for a future dev.

Reply:
Other than that it looks good to me